int32 simState
bool obj1update
bool obj2update
geometry_msgs/Point obj1initial
geometry_msgs/Point obj2initial
float64 obj1mass
float64 obj2mass
float64 spring_constant
float64 damper_constant
float64 equil_spring_length
float64 timeDelta
---
geometry_msgs/Point obj1final
geometry_msgs/Point obj2final